lib/rails/generators/haml/component_generator.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require "rails/generators/erb/component_generator"
+
+module Haml
+  module Generators
+    class ComponentGenerator < Erb::Generators::ComponentGenerator
```

Comment:
should we inherit from `Base` here too?

Reply:
It seems to be the norm to inherit from the corresponding Erb generator: https://github.com/haml/haml-rails/blob/master/lib/generators/haml/scaffold/scaffold_generator.rb#L5